### PR TITLE
update chinese docs 

### PR DESCRIPTION
**Pull request by lovesomeone**
1. Synchronize the ReadMe
2. translate  migration guides 
3. translate  some files in docs_CN

### CODE REVIEW

File: docs_CN/RxSwift.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# RxSwift
+# RxSwift（参照ReactiveSwift）
```
Comment (BasThomas) [inline docs_CN/RxSwift.md]: This is RxSwift, not ReactiveSwift. Why did you add this?

Comment (lovesomeone) [reply docs_CN/RxSwift.md]: because description is the same。so Translation is the same

**Comment by lovesomeone** [comment]
@BasThomas @stephencelis @steam @shanev  hi    Do I need to  click on "merge pull request" button（show below） 

 or wait for someone else to check the branch then make merge ?

2a4f094

**Comment by SD10** [comment]
Hey @lovesomeone, great question 😃

I usually wait for a couple Moya Core Team members to approve my pull requests before merging. Personally, I never merge my own PRs 😅 

It's good to get a few pairs of eyes on everything if possible. I know this is related to Chinese so reviewers are sparse, but @BasThomas was still able to give some feedback.

File: docs_CN/RxSwift.md (hunk 1 above)
Comment (SD10) [reply docs_CN/RxSwift.md]: @lovesomeone The descriptions may be similar but the syntax provided in the code snippets is different. I think that there should be a `RxSwift.md` & `ReactiveSwift.md` just like the English docs.

Comment (lovesomeone) [reply docs_CN/RxSwift.md]: Ok  you're right . It may be ambiguous.   I'll update it 。Keep them  separate。

**Comment by AndrewSB** [comment]
This is awesome :)
Can you delete c9981ee and rebase instead? I think it's ready to merge after that

**Comment by lovesomeone** [comment]
Hey  @AndrewSB  .   Because I  never used rebase on github，can you tell me how to do it.  Or can someone help to delete c9981ee and rebase. Thank you!

**Comment by AndrewSB** [comment]
@lovesomeone for sure!!

So you're going to do two things.
1. delete that commit
2. rebase your branch onto the latest Moya:master.

`git rebase` is a really interesting and often hard to understand tool, https://git-scm.com/book/en/v2/Git-Branching-Rebasing is a great resource if you're interested.

To delete the commit, you're going to do an interactive rebase (see https://stackoverflow.com/a/1338756)

so I think the command will be 

```
git rebase -i c9981ee~1
```

which will bring up your `$EDITOR`, where you should drop c9981ee and save. That should delete the commit

To rebase your changes onto Moya:master, first you'll need to have Moya/Moya as a git remote (you can see all your remotes with `git remote -v`), assuming you have it as a remote named `origin` that points to Moya/Moya, you can

```
git fetch origin
git rebase origin/master
```

and if that worked without error, you can force push to your fork (assuming the remote is named `lovesomeone`)

```
git push lovesomeone -f
```

let me know if I can clear anything up or if I was confusing anywhere!

**Comment by sunshinejr** [comment]
I assume this one can be closed now as #1365 went on.
